harvester_e2e_tests/fixtures/vm.py

```diff
@@ -60,12 +60,12 @@ def user_data_with_guest_agent(keypair):
       - %s
 package_update: true
 packages:
-  - qemu-guest-agent
+  - qemu-ga
```

Comment:
The package name is correct, it should be `qemu-guest-agent`. Only the service name need to be `qemu-ga`.

Reply:
done